agent/consul/fsm/commands_oss.go

```diff
@@ -378,61 +378,69 @@ func (c *FSM) applyConnectCAOperation(buf []byte, index uint64) interface{} {
 		[]metrics.Label{{Name: "op", Value: string(req.Op)}})
 	defer metrics.MeasureSinceWithLabels([]string{"fsm", "ca"}, time.Now(),
 		[]metrics.Label{{Name: "op", Value: string(req.Op)}})
+
+	result := ApplyConnectCAOperationFromRequest(c.state, &req, index)
+	if err, ok := result.(error); ok && err != nil {
+		c.logger.Warn("Invalid CA operation", "operation", req.Op)
```

Comment:
Is this warning appropriate for cases where the state store functions return an error? In that case the error isn't related to passing an invalid `req.Op`

Reply:
Oops, true. I will fix this to say "Failed to apply CA operation"